.github/workflows/module-tests.yml

```diff
@@ -80,6 +80,7 @@ jobs:
             !presto-redis,
             !presto-sqlserver,!presto-postgresql,!presto-mysql,
             !presto-phoenix,!presto-iceberg,
+            !presto-kudu,
```

Comment:
the exlcudes should be same order as the section below, so it's possible to eye-ball them together and catch diffs

-> move kudu to be last

Reply:
Actually, the `!presto-docs,!presto-server,!presto-server-rpm` line should be at the bottom, since it's special (it's not in the list below).